src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -746,4 +772,16 @@ public SnapshotSummary mapRow(ResultSet rs, int rowNum) throws SQLException {
           .selfHosted(rs.getBoolean("self_hosted"));
     }
   }
+
+  private static class UuidMapper implements RowMapper<UUID> {
```

Comment:
Shamelessly copy-pasted from elsewhere in the repo.

Reply:
If it's used in more than one place, can the code be shared instead of copied?

Reply:
Yep!  Removed duplicated code and updated references.